doc/build-with-cmake.md

```diff
@@ -0,0 +1,8 @@
+Introduction
+============
```

Comment:
Turns out I prefer these too, but I didn't know about them until now. Will change these in the next patch.